receiver/kafkareceiver/header_extraction.go

```diff
@@ -0,0 +1,77 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package kafkareceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver"
+
+import (
+	"fmt"
+
+	"github.com/Shopify/sarama"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	"go.uber.org/zap"
+)
+
+var getAttribute = func(key string) string {
+	return fmt.Sprintf("kafka.header.%s", key)
+}
+
+type headerExtractor struct {
+	logger  *zap.Logger
+	headers []string
+}
+
+func (ex *headerExtractor) extractHeadersTraces(traces ptrace.Traces, message *sarama.ConsumerMessage) {
```

Comment:
```suggestion
func (h *headerExtractor) extractHeadersTraces(traces ptrace.Traces, message *sarama.ConsumerMessage) {
```

Just a from thing

Reply:
typical I would suggest the first letter of each word  of the type like, `he`, but `h` is fine, but I agree with moving away from `ex`